systems/plants/RigidBodyFlatTerrain.m

```diff
@@ -3,13 +3,18 @@
 %  everywhere
   
   methods 
-    function obj = RigidBodyFlatTerrain()
+    function obj = RigidBodyFlatTerrain(varargin)
+      % Construct a flat terrain map at a given z height (default 0)
+      p = inputParser();
```

Comment:
my vote would be to stick with option structs instead of using inputParser, because:
 1) everywhere else in the code, we use an options struct (instead of 'z', ...)
 2) options structures have the benefit of being able to be passed through to submethods, etc, very easily, where this version of optional input does not.  (you could also say this is a negative, but we use it frequently).  

what do others think?